domain/status/modelmigration/export.go

```diff
@@ -64,28 +65,37 @@ func (e *exportOperation) Name() string {
 // Setup the export operation.
 // This will create a new service instance.
 func (e *exportOperation) Setup(scope modelmigration.Scope) error {
-	e.service = service.NewService(
-		state.NewState(scope.ModelDB(), e.clock, e.logger),
-		e.clock,
-		e.logger,
-		// TODO(jack): This is currently the wrong logger. We should construct
-		// the StatusHistory using the model logger, however, at the moment, we
-		// cannot get the model logger until the model has been imported. Once
-		// this has changed, refactor this to use the model logger.
-		domain.NewStatusHistory(e.logger, e.clock),
-	)
+	e.serviceGetter = func(modelUUID model.UUID) ExportService {
+		return service.NewService(
+			state.NewState(scope.ModelDB(), e.clock, e.logger),
+			modelUUID,
+			// TODO(jack): This is currently the wrong logger. We should construct
+			// the StatusHistory using the model logger, however, at the moment, we
+			// cannot get the model logger until the model has been imported. Once
+			// this has changed, refactor this to use the model logger.
+			domain.NewStatusHistory(e.logger, e.clock),
+			func() (service.StatusHistoryReader, error) {
+				return nil, errors.Errorf("status history reader not available")
```

Comment:
Do we still ship logs with migration?

Reply:
Yes, but we don't need to read them during migration.